docs/refmans/gui/viselements/generic/charts/advanced.md_template

```diff
@@ -442,9 +442,9 @@ selected_indices = []
 
 mean_value = 0.
 
-def on_change(state, var, val):
-    if var == "selected_indices":
-        state.mean_value = numpy.mean([data["y"][idx] for idx in val]) if len(val) else 0
+def on_change(state, var_name, var_value):
```

Comment:
We can use type hinting as much as possible in our examples, thereby spreading a good practice.